nylas/api/wsgi.py

```diff
@@ -53,6 +53,14 @@ def log_request(self):
         if request_uid is not None:
             additional_context['request_uid'] = request_uid
 
+        # pywsgi negates the status code if there was a socket error
```

Comment:
minor nit: this might be just the pywsgi gevent adapter, and not pywsgi in general. might be worth linking to the code you found for context in this comment.